src/types/mod.rs

```diff
@@ -10,36 +10,32 @@
 //! * Strings (`String` and `&str`)
 //! * Blobs (`Vec<u8>` and `&[u8]`)
 //!
-//! Additionally, because it is such a common data type, implementations are
-//! provided for `time::Timespec` that use the RFC 3339 date/time format,
+//! Additionally, if the `time` feature is enabled, implementations are
+//! provided for `time::OffsetDateTime` that use the RFC 3339 date/time format,
 //! `"%Y-%m-%dT%H:%M:%S.%fZ"`, to store time values as strings.  These values
 //! can be parsed by SQLite's builtin
 //! [datetime](https://www.sqlite.org/lang_datefunc.html) functions.  If you
-//! want different storage for timespecs, you can use a newtype. For example, to
-//! store timespecs as `f64`s:
+//! want different storage for datetimes, you can use a newtype. For example, to
+//! store datetimes as `i64`s counting the number of seconds since the Unix
+//! epoch:
 //!
-//! ```rust
+//! ```ignore
```

Comment:
Not tested, but it seems possible to activate feature:
https://users.rust-lang.org/t/doctests-that-require-a-non-default-feature-is-it-possible/29529/4

Reply:
Sure thing, done.